Todolist/index.js

```diff
@@ -0,0 +1,287 @@
+    refreshUI();
+    lists.forEach(list => {
+        selectList.innerHTML += `<option value="${list.id}" id="${list.text}"> ${list.text}</option>`;
+    });
+});
+
+function renderLists(){
+    lists.forEach(list => {
+        list.count = 0;
+        
+    });
+
+    todos.forEach(todo =>{
+        lists.forEach(list =>{
+            if(todo.day === list.text){
+                list.count++;
+            }
+        });
+    });
+
+    listsContainer.innerHTML = '';
+    lists.forEach(list =>{
+        listsContainer.innerHTML += renderListItem(list);
+    });
+}
+
+function renderListItem(list){
+    return `
+        <div class="list">
+            <h3>${list.text}</h3>
+            ${list.count} tareas
+        </div>
+    `;
+}
+
+function Todo (id,text,time,day,list,completed){
+    return {id:id,text:text,time:time,day:day,list:list,completed:completed};
+}
+
+addEventListener('submit',e =>{
+    e.preventDefault();
+    const text = input.value.trim();
+    const time = timePicker.value;
+    const listId = selectList.value;
+    const toDay = selectList.options[selectList.selectedIndex].id;
+    
+    if(text ==='') return false;
+
+    const newTodo = new Todo(uuid(), text, time, toDay, listId, false);
+
+    todos.push(newTodo);
+    localStorage.setItem('Todos',JSON.stringify(todos));
+    input.value = '';
+    timePicker.value= '';
+
+    refreshUI();
+});
+
+timePicker.addEventListener('keyup', function (event) {
+    if (event.key=='Enter'){
+        const text = input.value.trim();
+        const time = timePicker.value;
+        const listId = selectList.value;
+        const toDay = selectList.options[selectList.selectedIndex].id;
+    
+    if(text ==='') return false;
+
+        const newTodo = new Todo(uuid(), text, time, toDay, listId, false);
+
+        todos.push(newTodo);
+        localStorage.setItem('Todos',JSON.stringify(todos));
+        input.value = '';
+        timePicker.value= '';
+
+        refreshUI();
+    }
+});
+
+addTaskButton.addEventListener('click',e =>{
+    e.preventDefault();
+    const text = input.value.trim();
+    const time = timePicker.value;
+    const listId = selectList.value;
+    const toDay = selectList.options[selectList.selectedIndex].id;
+    
+    if(text ==='') return false;
+
+    const newTodo = new Todo(uuid(), text, time, toDay, listId, false);
+
+    todos.push(newTodo);
+    localStorage.setItem('Todos',JSON.stringify(todos));
+    input.value = '';
+    timePicker.value= '';
+
+    refreshUI();
+});
+
+function refreshUI(){
+    var d=selectList.selectedIndex;
+
+    renderTodos();
+    renderLists();
+
+    listFilter('Monday',hide,0);
+
+    if(d=== 0){
+        listFilter('Monday',hide,0);
+    }if(d=== 1){
+        listFilter('Tuesday',hide,1);
+    }if(d=== 2){
+        listFilter('Wednesday',hide,2);
+    }if(d=== 3){
+        listFilter('Thursday',hide,3);
+    }if(d=== 4){
+        listFilter('Friday',hide,4);
+    }  
```

Comment:
En vez de usar varios if's, puedes usar un switch case, en javascript sería algo así ;D

**[Esta es una forma de corregir en dónde te dejo escrito el cambio aquí mismo en el PR y luego tu puedes darle al botón "aplicar cambios" para que el cambio en tu codigo se haga solo y se suba un commit automaticamente ;D]**
```suggestion
    switch(d) {
  case 0:
    listFilter('Monday',hide,0);
    break;
  case 1:
    listFilter('Tuesday',hide,1);
    break;
  case 2:
    listFilter('Wednesday',hide,2);
    break;
  case 3:
    listFilter('Thursday',hide,3);
    break;
  case 4:
    listFilter('Friday',hide,4);
    break;
}
```